dotCMS/src/test/java/com/dotcms/rest/api/v1/site/SiteResourceTest.java

```diff
@@ -216,7 +216,7 @@ public void testSwitchNullEmptyAndInvalidFilter() throws JSONException, DotSecur
         when(request.getSession(false)).thenReturn(session);
         when(session.getAttribute(Globals.LOCALE_KEY)).thenReturn(new Locale.Builder().setLanguage("en").setRegion("US").build());
         SiteBrowserResource siteBrowserResource =
-                new SiteBrowserResource(webResource, new SiteBrowserHelper( hostAPI ), I18NUtil.INSTANCE, userAPI);
+                new SiteBrowserResource(webResource, new SiteHelper( hostAPI ), I18NUtil.INSTANCE, userAPI);
```

Comment:
I think this test won't compile. The resource was just renamed in this Pull Request to SiteResource.